platforms/ios/src/TangramMap/TGMapViewController.mm

```diff
@@ -135,7 +136,13 @@ - (void)loadSceneFile:(NSString *)path sceneUpdates:(NSArray<TGSceneUpdate *> *)
     }
 
     self.scenePath = path;
-    self.map->loadScene([path UTF8String], false, updates);
+
+    auto updateCallbackStatus = [self](auto sceneUpdateStatuses) {
```

Comment:
Hmmmm this looks like a closure called from C++ yeah? If so, you'll want to do the weakSelf/strongSelf dance to avoid a retain cycle with self. You can see a discussion on the topic here - https://github.com/objc-zen/objc-zen-book#retain-cycles-on-self

I tend to err on the side of caution here - there are times when you don't have to do this dance because everything is behaving and not holding onto long-term references to the block or the code using the block. However it's a safeguard that I use a lot.